ocs_ci/ocs/platform_nodes.py

```diff
@@ -290,6 +290,9 @@ def restart_nodes(self, nodes, force=True, timeout=300, wait=True):
 
         self.vsphere.restart_vms(vms, force=force)
 
+        logger.info(f"Waiting for 30 seconds...")
+        time.sleep(30)
+
```

Comment:
Do not add extra wait time if wait=False

Reply:
Done